cl_billing_pricing.rules_default.inc

```diff
@@ -1,46 +0,0 @@
-<?php
```

Comment:
In our use case, it will modify the price twice. I believe it would be better to either document how to create the rule (in README file) or to find a way to prevent processing line items twice. Ideas?